### PR TITLE
Code review request

### PR DESCRIPTION
**Pull request by jakobotten**
check formatting, naming and logic @patdriscoll61

### CODE REVIEW

**Comment by jakobotten** [comment]
@patdriscoll61

File: Prac08/test_taxi.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+from Prac08.taxi import Taxi
```
Comment (ghost) [inline Prac08/test_taxi.py:1]: May be a comment string to explain the program??

**Review by ghost — COMMENTED** [review]
Cheers

File: Prac08/test_taxi.py (hunk 1 above)
Comment (jakobotten) [reply Prac08/test_taxi.py:1]: Thanks for the feedback.

**Pull request merged**